lib/mayaUsd/ufe/UsdStageMap.cpp

```diff
@@ -182,7 +182,8 @@ MObject UsdStageMap::proxyShape(const Ufe::Path& path)
                 // Key is stale.  Remove it from our cache, and add the new entry.
                 auto count = fPathToObject.erase(cachedPath);
                 TF_AXIOM(count);
-                fPathToObject[newPath] = cachedObject;
+                if (!newPath.empty())
```

Comment:
When undoing delete of stages, it sometimes occurred that the newPath was empty and caused a crash when used later on. Avoid filling the stage map with invalid paths.